console/v1/types_console_quick_start.go

```diff
@@ -0,0 +1,126 @@
+package v1
+
+import (
+	authorizationv1 "k8s.io/api/authorization/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConsoleQuickStart is an extension for guiding user through various
+// workflows in the OpenShift web console.
+type ConsoleQuickStart struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// +kubebuilder:validation:Required
+	// +required
+	Spec ConsoleQuickStartSpec `json:"spec"`
+}
+
+// ConsoleQuickStartSpec is the desired quick start configuration.
+type ConsoleQuickStartSpec struct {
+	// displayName is the display name of the Quick Start.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	DisplayName string `json:"displayName"`
+	// icon is a base64 encoded image that are used as Quick Start's
+	// +optional
+	Icon string `json:"icon,omitempty"`
+	// tags is an array of strings that describe the Quick Start.
+	// +optional
+	Tags []string `json:"tags,omitempty"`
+	// durationMinutes describes approximately how many minutes it will take to complete the Quick Start.
+	// +optional
+	DurationMinutes int `json:"durationMinutes,omitempty"`
+	// description is the description of the Quick Start. (includes markdown)
+	// +optional
+	Description string `json:"description,omitempty"`
+	// prerequisites contains a list of Quick Start names that must be completed before this Quick Start.
+	// +optional
+	Prerequisites []string `json:"prerequisites,omitempty"`
+	// introduction describes the purpose of the Quick Start. (includes markdown)
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Introduction string `json:"introduction"`
+	// tasks is the list of steps the user has to perform to complete the Quick Start.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinItems=1
+	// +required
+	Tasks []ConsoleQuickStartTask `json:"tasks"`
+	// conclusion sums up the Quick Start and suggests the possible
+	// next Quick Starts for the user to try. (includes markdown)
+	// +optional
+	Conclusion string `json:"conclusion,omitempty"`
```

Comment:
`nextQuickStart` is used to suggest the next quick starts for the user to try.

Reply:
The will rephrase it to `conclusion sums up the Quick Start and suggests the possible steps. (includes markdown)`